Firebase/Messaging/FIRMessagingContextManagerService.m

```diff
@@ -44,6 +44,8 @@
 NSString *const kFIRMessagingContextManagerSoundKey = kFIRMessagingContextManagerNotificationKeyPrefix @"sound";
 NSString *const kFIRMessagingContextManagerContentAvailableKey =
     kFIRMessagingContextManagerNotificationKeyPrefix @"content-available";
+NSString *const kFIRMessagingAPNSPayload = @"aps";
```

Comment:
Should this be `kFIRMessagingAPNSPayloadKey`?

Reply:
Done!